README.md

```diff
@@ -8,10 +8,10 @@
 
 ## Installation 
 
-### One-click Known sites
+### Shared Host on  Known sites
 
-If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013),
-which includes one-click Known installation. Known is also known to work on [DreamHost](https://dreamhost.com), a high-quality
+If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation.The one click installs an outdated version of 0.9.2 and then requires updating following the intructions below.  
```

Comment:
```suggestion
If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation. Currently this installs version 0.9.2, which then requires updating using the instructions below.  
```

Reply:
Interestingly, GitHub gives me the button to merge these changes, but gives me a permission error when actually attempting to do so

Reply:
I get the same. It's because we don't own @jgmac1106's branch. He needs to merge them!

Reply:
Bit daft to show the button if you don't have access to perform the action. Basic UX